docker-compose.yml

```diff
@@ -7,4 +7,6 @@ services:
       - "3306:3306"
     environment:
       MYSQL_ROOT_PASSWORD: password
-      MYSQL_PASSWORD: password
\ No newline at end of file
+      MYSQL_PASSWORD: password
+    volumes:
+      - C:\Users\adamc\code\jmyapi\docker\99_busy.sql.gz:/docker-entrypoint-initdb.d/99_busy.sql.gz
```

Comment:
Having your home directory path in the docker-compose will break it for everyone else.    Use a path relative to the project root.    Sounds like this compressed test data will need to be committed to the repo.

Reply:
Fixed.  Simple oversight.